plugins/flashrom/fu-flashrom-plugin.c

```diff
@@ -100,6 +100,8 @@ fu_flashrom_plugin_device_set_bios_info(FuPlugin *plugin, FuDevice *device, GErr
 	gsize bufsz;
 	guint32 bios_char = 0x0;
 	g_autoptr(GPtrArray) bios_tables = NULL;
+	guint8 bios_sz = 0x0;
+	guint64 firmware_size = 0x0;
```

Comment:
can these two move into the `if` block please -- then LGTM -- thanks.

Reply:
Ohh they need to be at the top of the `{` - just nested a bit deeper. Thanks.

Reply:
done, I think :)